### ISSUE TITLE
Conformer generator produces strange structure for substituted butadiene

### ISSUE THREAD

**Issue by gncs** [rdkit/rdkit#5913]
**Describe the bug**

The conformer generator produces a strange structure for substituted butadiene.

**To Reproduce**

```python
from rdkit.Chem import AllChem

params = AllChem.SmilesParserParams()
params.removeHs = False
mol = AllChem.MolFromSmiles("[H]/C(F)=C([H])\C([H])=C(/[H])Br", params)

AllChem.EmbedMultipleConfs(mol, randomSeed=2, numConfs=10)

dihedral = (2, 1, 3, 5)
print(mol.GetBondBetweenAtoms(dihedral[1], dihedral[2]).GetStereo())
print([f'{AllChem.GetDihedralDeg(mol.GetConformer(conf_id), *dihedral):.1f}'
       for conf_id in range(mol.GetNumConformers())])
```

Output:
```text
STEREOE
['-180.0', '-180.0', '180.0', '-7.2', '180.0', '180.0', '-180.0', '180.0', '180.0', '180.0']
```

**Expected behavior**

I would expect all conformers to look like this:

<img width="302" alt="Screenshot_20230101_045710" src="https://user-images.githubusercontent.com/1926852/210177396-a10d0391-288d-4726-9014-a670d6c3f7e6.png">

However, the fourth conformer looks like this:

<img width="307" alt="Screenshot_20230101_045647" src="https://user-images.githubusercontent.com/1926852/210177418-23243369-1498-4b90-be69-2c6bc7463850.png">

**Configuration**
 - RDKit version: 2022.09.1, py38h5acb366_1, conda-forge
 - OS: Ubuntu 20.04

**Comment by greglandrum** [comment]
Hi @gncs, thanks for the detailed bug report.
This is one that we are aware of and hope to have fixed in a not-too-distant RDKit release.

**Comment by greglandrum** [comment]
This was fixed in #5967. It will be in the 2023.03.1 release.
